src/Microsoft.ML.Transforms/Microsoft.ML.Transforms.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.1</TargetFrameworks>
```

Comment:
You are going to need to do a lot more work here in order to get our nuget package to be multi-targeted correctly.

NuGet only picks assets from 1 TFM. And since we put many assemblies in `Microsoft.ML`, you are only making this assembly multi-targeted. Which means the NuGet package will have all assemblies in `netstandard2.0` and only this assembly in `netcoreapp2.1`. So if a consumer is targeting netcoreapp2.1, they won't get references to all the other assemblies.

We already have a double parser code in ML.NET that takes in a `ROS<char>`, maybe you can change your code to use that so you don't have to take a dependency on `netcoreapp2.1`-specific APIs?

Reply:
@eerhardt  I had no idea about the existence of DoubleParser. I have re-used it and now we target just .NET Standard 2.0 again.